server/src/main/java/io/druid/segment/indexing/DataSchema.java

```diff
@@ -58,6 +59,7 @@ public DataSchema(
       @JsonProperty("parser") Map<String, Object> parser,
       @JsonProperty("metricsSpec") AggregatorFactory[] aggregators,
       @JsonProperty("granularitySpec") GranularitySpec granularitySpec,
+      @JsonProperty("rollup") Boolean rollup,
```

Comment:
this isn't really a property of the dataschema and should be a part of the TuningConfig